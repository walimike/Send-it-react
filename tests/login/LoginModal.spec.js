import React from 'react';
import { shallow } from 'enzyme';

import LoginModal from '../../src/components/login/loginModal';

describe('LoginModal', () => {
  it('should render without crashing', () => {
    const wrapper = shallow(<LoginModal />);
    expect(wrapper).toMatchSnapshot();
  });
});
