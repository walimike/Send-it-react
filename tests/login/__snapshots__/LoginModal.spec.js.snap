// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginModal should render without crashing 1`] = `ShallowWrapper {}`;
